cosmo_tester/test_suites/image_based_tests/cfy_manager_test.py

```diff
@@ -14,13 +14,15 @@
 GET_MQ_PASSWORDS_CODE_PATH = '/tmp/get_passwords.py'
 MQ_PASSWORDS_PATH = '/tmp/passwords'
 GET_MQ_PASSWORDS_CODE = '''
+import os
 import json
 
 from cloudify.cryptography_utils import decrypt
 
 from manager_rest.storage import models
 from manager_rest.flask_utils import setup_flask_app
 
+os.environ['MANAGER_REST_CONFIG_PATH'] = '/opt/manager/cloudify-rest.conf'
```

Comment:
This shouldn't be in the tests

Reply:
Without this line, it fails, so I wonder what would be the issue?

Reply:
That doesn't make any sense- there is no /opt/manager/cloudify-rest.conf on the test VM- and this doesn't get exported to the remote nodes.

Reply:
Marking this conversation as resolved- I didn't notice that this was actually in the GET_PASSWORDS code variable. D'oh.